src/test/java/seedu/address/testutil/TypicalBinItems.java

```diff
@@ -0,0 +1,59 @@
+package seedu.address.testutil;
+
+import static seedu.address.logic.commands.CommandTestUtil.VALID_COVERAGE_CAR_INSURANCE;
+import static seedu.address.logic.commands.CommandTestUtil.VALID_DATE_DELETED_CAR_INSURANCE;
+import static seedu.address.logic.commands.CommandTestUtil.VALID_DATE_DELETED_DAVID;
+import static seedu.address.logic.commands.CommandTestUtil.VALID_DATE_DELETED_KASSANDRA;
+import static seedu.address.logic.commands.CommandTestUtil.VALID_DATE_DELETED_LINCOLN;
+import static seedu.address.logic.commands.CommandTestUtil.VALID_DESCRIPTION_CAR_INSURANCE;
+import static seedu.address.logic.commands.CommandTestUtil.VALID_END_AGE_CAR_INSURANCE;
+import static seedu.address.logic.commands.CommandTestUtil.VALID_EXPIRY_DATE_CAR_INSURANCE;
+import static seedu.address.logic.commands.CommandTestUtil.VALID_EXPIRY_DATE_DAVID;
+import static seedu.address.logic.commands.CommandTestUtil.VALID_EXPIRY_DATE_KASSANDRA;
+import static seedu.address.logic.commands.CommandTestUtil.VALID_EXPIRY_DATE_LINCOLN;
+import static seedu.address.logic.commands.CommandTestUtil.VALID_NAME_CAR_INSURANCE;
+import static seedu.address.logic.commands.CommandTestUtil.VALID_PRICE_CAR_INSURANCE;
+import static seedu.address.logic.commands.CommandTestUtil.VALID_START_AGE_CAR_INSURANCE;
+
+import java.util.ArrayList;
+import java.util.Arrays;
+import java.util.List;
+
+import seedu.address.commons.util.BinItemBuilder;
+import seedu.address.commons.util.PersonBuilder;
+import seedu.address.commons.util.PolicyBuilder;
+import seedu.address.model.binitem.BinItem;
+
+/**
+ * A utility class containing a list of {@code BinItem} objects to be used in tests.
+ */
+public class TypicalBinItems {
+
+    public static final BinItem BIN_DAVID = new BinItemBuilder().withItem(new PersonBuilder().withName("David Georgia")
```

Comment:
Best prac is to avoid magic string literals. Could have imported from `PersonBuilder`.

Reply:
Ohh, I think you mean `CommandTestUtil` right. Alright I'll shift them there.